acquisitions/src/main/resources/thunderjet/consortia/reusable/enableInstanceMatching.feature

```diff
@@ -0,0 +1,20 @@
+@ignore
+Feature: Enable Instance Matching
+
+  Background:
+    * url baseUrl
+
+  Scenario: enableInstanceMatching
+    Given path 'configurations/entries'
+    And param query = 'configName==disableInstanceMatching'
+    When method GET
+    Then status 200
+    * def config = $.configs[0]
+    * set config.value = "{\"isInstanceMatchingDisabled\":false}"
+    * def configId = $.configs[0].id
+
+    Given path 'configurations/entries', configId
+    And request config
+    When method PUT
+    Then status 204
+    * call pause 40000
```

Comment:
when implement story to replace mod-configuration usage please introduce new header to bypass caching, it will allow to not wait so long in this test and other similar